doc/api.yml

```diff
@@ -1165,7 +1205,11 @@ components:
     PageSize:
       type: integer
       description: Cannot exceed `500`
-
+    CourseDirection:
```

Comment:
No.

Leave the OrderDirection.

Just inherit the CourseDirection with OrderDirection.